conda/env/env.py

```diff
@@ -291,3 +696,22 @@ def print_result(args, prefix, result):
             common.stdout_json_success(prefix=prefix, actions=actions)
     else:
         install.print_activate(args.name or prefix)
+
+
+def get_env_name(prefix: os.PathLike) -> str:
```

Comment:
I'm sure there's functionality for this somewhere else in the code. Maybe not as a function, but I've seen this logic elsewhere. I think I'll address this as part of https://github.com/conda/conda/pull/14750, though, so let's leave this thread open.